okhttp/third_party/okhttp/java/io/grpc/okhttp/internal/OptionalMethod.java

```diff
@@ -169,6 +169,12 @@ private Method getMethod(Class<?> clazz) {
   private static Method getPublicMethod(Class<?> clazz, String methodName, Class[] parameterTypes) {
     Method method = null;
     try {
+      if(clazz == null) {
```

Comment:
```getSuperclass()``` returns ```null``` for Object or other base types that don't have parents, so this serves as the base case.

I've fixed style issue.